worker/awsCertlint/awsCertlint.go

```diff
@@ -0,0 +1,167 @@
+package awsCertlint
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/mozilla/tls-observatory/certificate"
+	"github.com/mozilla/tls-observatory/worker"
+)
+
+var (
+	workerName = "awsCertlint"
+	workerDesc = "Runs awslabs/certlint over a given Certificate, categoriezes output for display on the certificate"
+
+	certlintDirectory = "/go/certlint" // path from tools/Dockerfile-scanner
```

Comment:
This path isn't _ideal_, but is it worthwhile to change? Probably, since I assume we're wanting to add more binaries to check certificates with.

Reply:
This should be configurable, similar to what we did with evChecker https://github.com/mozilla/tls-observatory/blob/master/worker/evCheckerWorker/evCheckerWorker.go#L24